recipes/bioconductor-rhtslib/meta.yaml

```diff
@@ -21,9 +23,15 @@ requirements:
   host:
     - 'bioconductor-zlibbioc >=1.28.0,<1.29.0'
     - r-base
+    - zlib
+    - xz
+    - bzip2
   run:
     - 'bioconductor-zlibbioc >=1.28.0,<1.29.0'
     - r-base
+    - zlib
```

Comment:
just for your information, all those should have a `run_exports` defined and no not need to be mentioned in the `run` section anymore.

Reply:
Ah, right